crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -98,7 +98,7 @@ impl Display for DisplayRepresentation<'_> {
                 let escape =
                     AsciiEscape::with_preferred_quote(bytes.value(self.db).as_ref(), Quote::Double);
 
-                escape.bytes_repr().write(f)
+                escape.bytes_repr(false).write(f)
```

Comment:
Because Rust doesn't support keyword arguments, it can be a little unreadable to have functions that take boolean parameters. At a callsite like here, it can be hard to tell why a `bool` is being passed into a function. It's often nicer to create a custom enum that can be passed into the function, e.g.

```rs
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
enum TripleQuoted {
    Yes,
    No,
}
```

And then if the parameter took an instance of this enum, this callsite could be

```suggestion
                escape.bytes_repr(TripleQuoted::No).write(f)
```